app/clients/freshdesk.py

```diff
@@ -1,12 +1,84 @@
 import json
-
 import requests
+
 from requests.auth import HTTPBasicAuth
+from typing import Dict, List, Union
 
 from flask import current_app
 
+from app.user.contact_request import ContactRequest
+
+
+__all__ = [
+    'Freshdesk'
+]
```

Comment:
Needed?

Reply:
same as the other comment, it's good practice to include public and private interfaces based upon the pep-0008 recommendations https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces